integration-tests/shard-operator-integration-tests/shard-operator-integration-tests-v2/src/test/resources/features/processor-resource.feature

```diff
@@ -0,0 +1,23 @@
+Feature: ManagedProcessor tests
+
+  Background:
+    Given create Namespace
+
+  @VIP
+  Scenario: ManagedProcessor resource
+    When deploy ProcessorResource:
+    """
+      apiVersion:  com.redhat.service.smartevents/v2alpha1
+      kind: ManagedProcessor
+      metadata:
+        name: "processor-name-metadata"
+      spec:
+        name: "processor-name"
+        flows:
+          - from:
+              uri: "rhose:bridge"
+              steps:
+              - to:
+                  uri: "sink-name"
+    """
+    Then the ProcessorResource "processor-name" exists within 1 minute
```

Comment:
The name of the processor does not match. "processor-name-metadata" != "processor-name"

Reply:
Thank you there are two fields "name" one in **metadata** and one in **spec**. I was not sure which one is valid:
```
...
      metadata:
        name: "processor-name-metadata"
      spec:
        name: "processor-name"
...
```

Reply:
I think that metadata -> name is important for the fabric8 client to find the resource. Name in spec is only the part of specification and in theory it may not be unique.

Reply:
metadata/name is a mandatory parameter for every Kuberenetes resource, serves as an identifier of a resource (in combination with resource type)